### PR TITLE
Use NPM 3.3.6 if the default node is 4.1.2 to workaround path issues …

### PR DESCRIPTION
**Pull request by ahmelsayed**
…with aspnet 5

### CODE REVIEW

File: Kudu.Core/Infrastructure/PathUtility.cs [hunk 1]
```diff
@@ -136,10 +136,23 @@ private static string ResolveNpmVersion()
         private static string ResolveNpmVersion(string nodeVersion)
         {
             string programFiles = SystemEnvironment.GetFolderPath(SystemEnvironment.SpecialFolder.ProgramFilesX86);
+            string appSettingNpmVersion = SystemEnvironment.GetEnvironmentVariable("APPSETTING_WEBSITE_NPM_DEFAULT_VERSION");
```
Comment (davidebbo) [inline Kudu.Core/Infrastructure/PathUtility.cs]: Just `WEBSITE_NPM_DEFAULT_VERSION` should work too and is cleaner for the non-Azure case.

**Comment by davidebbo** [comment]
Added to doc: https://github.com/projectkudu/kudu/wiki/Configurable-settings#change-the-npm-version

**Comment by davidebbo** [comment]
Looks good other than minor comment.

**Comment by ahmelsayed** [comment]
Updated and merged. Thanks!

**Pull request closed without merging**